modules/data.atmosphere/R/met.process.R

```diff
@@ -244,7 +244,7 @@ met.process <- function(site, input_met, start_date, end_date, model,
           end_date <= end_date,
           format_id == formatid
         ) %>%
-        filter(grepl(met, name)) %>%
+        filter(grepl(met, "name")) %>%
```

Comment:
Could you explain this change too? Are we really looking for the literal string "name" in the met, not the contents of the variable `name`?